Ducky-Chickens/bangazonCLI
# Tac jsdoc

# Description
Add jsdoc example and install it.

## Related Ticket(s)
Complete Issue #14 (Add Jsdoc and example.)

## Expected Behavior
An /out/ folder should exist in the root directory.

## Steps to Test Solution
1. Check that the folder exists and when you go to the index, the standard jsdoc documentation loads.
